docs/functions.rst

```diff
@@ -1572,6 +1572,67 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _backupdatausingkopiaserver:
+
+BackupDataUsingKopiaServer
+--------------------------
+
+This function backs up data from a container into any object store
+supported by Kanister using Kopia Repository Server as a data mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that capture path on a volume and store it on the object store.
+
+   Additionally, in order to use this function, a RepositoryServer CR is required
+
+Arguments:
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace of the application that you want to backup the data of
+   `pod`, Yes, `string`, pod of the application that you want to backup the data of
+   `container`, Yes, `string`, name of the kanister sidecar container
+   `includePath`, Yes, `string`, path of the data to be backed up
+   `userPassphrase`, Yes, `string`, user access credentials for kopia repository server
+   `certData`, Yes, `string`, certificate data for kopia repository server
+   `snapshotTags`, No, `string`, custom tags to be provided for the kopia snapshots
```

Comment:
```suggestion
   `namespace`, Yes, `string`, namespace of the container that you want to backup the data of
   `pod`, Yes, `string`, pod name of the container that you want to backup the data of
   `container`, Yes, `string`, name of the kanister sidecar container
   `includePath`, Yes, `string`, path of the data to be backed up
   `userPassphrase`, Yes, `string`, user access credentials for kopia repository server
   `certData`, Yes, `string`, certificate data for kopia repository server
   `snapshotTags`, No, `string`, custom tags to be provided for the kopia snapshots
```


Sorry, I think I should have been clear before. Since in the function desc here in the docs, we are saying it's used to backup the data from a container. Let's use that container's reference while describing the arguments.